qualtran/bloqs/bookkeeping/auto_partition.py

```diff
@@ -40,6 +48,8 @@ class AutoPartition(Bloq):
         bloq: The bloq to wrap.
         partitions: A sequence of pairs specifying each register that the wrapped bloq should accept
             and the names of registers from `bloq.signature.lefts()` that concatenate to form it.
+            If the bloq being wrapped does not use a portion of the register being exposed, an
+            instance of `Unused` may be used in place of a register name from the bloq signature.
```

Comment:
This docstring writ large is getting a little unwieldy if you don't already know what this bloq does. I've also noticed that the class docstring doesn't match the format of "short one-line summary, followed by details." My suggested text follows

```
"""Automatically partitions and unpartitions registers to match the signature of a sub-bloq

This can be used for adapting factored registers into a 'quantum interface'. Wrapping in AutoPartition
hides the partitioning splits and joins behind a level of decomposition, which can produce more
helpful circuit diagrams.

Args:
  bloq: The sub-bloq to wrap. Its register names are used as the 
    second items in the `partitions` tuples
  partitions: A sequence of pairs specifying how to partition each exterior register. The first element of 
    each pair is the `Register` that is exposed as part of AutoPartition's exterior signature. The 
    second element is a list of register names from the original subbloq, that, when combined, form the
    exposed register. If a portion of the exposed register is unused by the original subbloq, 
    the sentinel value `Unused(n)` can be used in place of a register name.

...
"""

Reply:
a suggestion ^ feel free to further revise

Reply:
Thanks! I revised the docs a little further.